src/extension.ts

```diff
@@ -202,38 +202,7 @@ export async function activate(ctx: vscode.ExtensionContext) {
   );
 
   if (!checkMesonIsConfigured(buildDir)) {
-    let configureOnOpen = configurationChosen || extensionConfiguration(SettingsKey.configureOnOpen);
-    if (configureOnOpen === "ask") {
-      enum Options {
-        yes = "Yes",
-        always = "Always",
-        no = "No",
-        never = "Never",
-      }
-
-      const response = await vscode.window.showInformationMessage(
-        "Meson project detected in this workspace but does not seems to be configured. Would you like VS Code to configure it?",
-        ...Object.values(Options),
-      );
-
-      switch (response) {
-        case Options.no:
-          break;
-
-        case Options.never:
-          extensionConfigurationSet(SettingsKey.configureOnOpen, false, vscode.ConfigurationTarget.Workspace);
-          break;
-
-        case Options.yes:
-          configureOnOpen = true;
-          break;
-
-        case Options.always:
-          extensionConfigurationSet(SettingsKey.configureOnOpen, true, vscode.ConfigurationTarget.Workspace);
-          configureOnOpen = true;
-          break;
-      }
-    }
+    let configureOnOpen = await askConfigureOnOpen();
```

Comment:
Seems like this can be const too?

Reply:
Was already changed in subsequent commits.